paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -230,13 +239,19 @@ class PaparazziPlugin : Plugin<Project> {
         test.systemProperties["paparazzi.snapshot.dir"] = snapshotOutputDir.toString()
         test.systemProperties["paparazzi.artifacts.cache.dir"] = gradleUserHomeDir.path
         test.systemProperties["kotlinx.coroutines.main.delay"] = true
+        test.systemProperties["app.cash.paparazzi.legacy.resource.loading"] = legacyResourceLoadingEnabled.toString()
+        test.systemProperties["app.cash.paparazzi.legacy.asset.loading"] = legacyAssetLoadingEnabled.toString()
```

Comment:
I'm assuming that not setting these was an omission

Reply:
Can you elaborate more?  Renderer appears to be able to see these system properties without this.

Reply:
That's very very strange, I'm not sure how it can. If you search the project for `app.cash.paparazzi.legacy.asset.loading` there is only a single result (the constant) and that constant is only used in a single place too.